l10n_it_pos_fatturapa/static/src/xml/pos.xml

```diff
@@ -0,0 +1,119 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<templates id="template" xml:space="preserve">
+
+    <t t-extend="ClientDetails">
+        <t t-jquery=".client-name" t-operation="after">
+            <div class="client-details-box clearfix">
+                <div class="client-detail">
+                    <span class='label'>Company</span>
+                    <input type='checkbox' class='detail is_company' name='is_company'
+                           t-att-checked="partner.is_company ? 'checked' : null" disabled="disabled"
+                           t-att-value="partner.is_company"/>
+                </div>
+            </div>
+        </t>
+
+        <t t-jquery=".client-details-left" t-operation="prepend">
+            <t t-if="partner.is_company == false">
+                <div t-attf-style="display: #{!partner.is_company ? 'block': 'none'};">
+                    <div class="client-detail">
+                        <span class='label'>First Name</span>
+                        <span class='detail firstname'><t t-esc='partner.firstname or ""'/></span>
+                    </div>
+                </div>
+            </t>
+        </t>
+
+        <t t-jquery=".client-details-right" t-operation="prepend">
+            <t t-if="partner.is_company == false">
+                <div t-attf-style="display: #{!partner.is_company ? 'block': 'none'};">
+                    <div class="client-detail">
+                        <span class='label'>Last Name</span>
+                        <span class='detail lastname'><t t-esc='partner.lastname or ""'/></span>
+                    </div>
+                </div>
+            </t>
+        </t>
+
+        <t t-jquery=".client-details-right" t-operation="append">
+            <div class='client-detail'>
+                <t t-if="partner.electronic_invoice_subjected">
+                    <span class='label'>e-Invoice</span>
+                    <input type="checkbox" class='detail o_checkbox electronic_invoice_subjected'
+                           t-att-checked="partner.electronic_invoice_subjected ? 'checked' : null" disabled="disabled"
+                           t-att-value="partner.electronic_invoice_subjected"/><br/>
+
+                    <div id="electronic_invoice_subjected" t-attf-style="display: #{partner.electronic_invoice_subjected ? 'block': 'none'};">
+                        <div class="client-detail">
+                            <span class='label'>Code</span>
+                            <span class='detail codice_destinatario'><t t-esc='partner.codice_destinatario or ""'/></span><br/>
+
+                            <span class='label'>PEC</span>
+                            <span class='detail pec_destinatario'><t t-esc='partner.pec_destinatario or ""'/></span><br/>
+
+                            <span class='label'>EORI</span>
+                            <span class='detail eori_code'><t t-esc='partner.eori_code or ""'/></span>
```

Comment:
```suggestion
                            <span class='detail eori_code'><t t-esc='partner.eori_code or ""'/></span><br/>
```
per correggere la visualizzazione in sola lettura (PA Code non va a capo):
![image](https://user-images.githubusercontent.com/32064796/69947132-4c695e80-14ed-11ea-9681-46cbcfe59ce7.png)